examples/year_playlist.py

```diff
@@ -0,0 +1,145 @@
+#!/usr/bin/env python3
+
+"""Script for retrieving songs from a user's 'Your Top Songs 2017'
+playlist.
+
+Note: Requests to Spotify API do not currently include relevant fields arg,
+which should eventually be added.
+"""
+
+from collections import namedtuple
+import sys
+import argparse
+import spotipy
+from spotipy import oauth2
+
+# Client ID and Client Secret for zachthehammer's spotipy-test API key
+_CLIENT_ID     = '5c243ed8edc9475a90bda2cc4a9a828c'
+_CLIENT_SECRET = 'b414caddc10143eea70db6edc1f4fb47'
+
+# Your Top Songs 2017 is actually owned by spotify user.
+_SPOTIFY_USER_ID    = 'spotify'
+_YEAR_PLAYLIST_NAME = 'Your Top Songs 2017'
+
+# Format strings
+_SCRIPT_DESCRIPTION_FMT = 'Get the list of tracks from a user\'s "{}" playlist.'
+_NO_PLAYLISTS_FOUND_FMT = 'No playlists found for user {}.'
+_PLAYLIST_NOT_FOUND_FMT = 'User "{user}"" does not follow their "{playlist}" playlist.'
+_USER_PLAYLIST_FMT      = '{user}\'s "{playlist}" playlist!'
+_TRACK_FMT              = '{artist} // {name} [{album}]'
+
+SpotifyTrack = namedtuple('SpotifyTrack', ['name', 'artist', 'album'])
+"""Namedtuple for spotify tracks."""
```

Comment:
usually you only have docstrings within: files, classes, or functions

Reply:
Technically a namedtuple is a class you're creating so I added a docstring. But seems from a quick search that that's not the convention.